.github/workflows/weekly_pull_requests.yaml

```diff
@@ -6,7 +6,10 @@ on:
         -   cron: "0 0 * * 0"
 
 jobs:
-    daily_pull_requests:
+    weekly_pull_requests:
```

Comment:
renamed the job.. I think thats a copy/paste leftover

Reply:
Thanks :+1: